samigo/samigo-app/src/webapp/jsf/delivery/noLateSubmission.jsp

```diff
@@ -43,10 +43,7 @@
   <h:outputText value="#{deliveryMessages.no_late_submission_url}" rendered="#{delivery.actionString=='takeAssessmentViaUrl'}"/>
   <p></p>
   <h:panelGroup  rendered="#{delivery.publishedAssessment.assessmentAccessControl.dueDate!=null}">
-    <h:outputLabel value="#{deliveryMessages.due_date}: " />
-	<h:outputText value="#{delivery.publishedAssessment.assessmentAccessControl.dueDate}" >
-		<f:convertDateTime dateStyle="medium" type="both" locale="#{UserLocale.locale}" timeZone="#{UserTimeZone.userTimeZone}"/>
-	</h:outputText>
+    <h:outputLabel value="#{deliveryMessages.due_date}: #{delivery.dayDueDateString}." />
```

Comment:
Is this dayDueDateString still localized for the user's preferred time zone?

Reply:
Yeah, it uses the TimeUtil class for getting the right timezone and I think it also checks the locale for the pattern.